src/xunit.performance.api/Profilers/Etw/AddressSpace.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Xunit.Performance.Api.Profilers.Etw
+{
+    /// <summary>
+    /// Represents an address space as given by the Microsoft.Diagnostics.Tracing.Parsers
+    /// </summary>
+    internal sealed class AddressSpace : ISpan<ulong>
+    {
+        /// <summary>
+        /// Initializes a new instance of the AddressSpace class.
+        /// </summary>
+        /// <param name="address">Interval's start address.</param>
+        /// <param name="size">Interval's size.</param>
+        public AddressSpace(ulong address, int size)
```

Comment:
Same note as below about size being unsigned.

Reply:
Same as below

---
In reply to: [145194737](https://github.com/Microsoft/xunit-performance/pull/242#discussion_r145194737) [](ancestors = 145194737)